client/src/dropzone-5.7.0/.gitignore

```diff
@@ -0,0 +1,7 @@
+build
```

Comment:
i think you're importing the entire `dropzone` folder in here which can be found here
https://github.com/enyo/dropzone

rather than uploading the entire package in here, we want to add this to package.json and install it in the environment

Reply:
I'm also not even using it.  So I'll just delete it for now